CefSharp.Wpf/Internals/IMEHandler.cs

```diff
@@ -15,9 +16,19 @@ namespace CefSharp.Wpf.Internals
     public static class ImeHandler
     {
         // Black SkColor value for underline.
-        internal const uint ColorUNDERLINE = 0xFF000000;
+        internal static uint ColorUNDERLINE = 0xFF000000;
         // White SkColor value for background.
-        internal const uint ColorBKCOLOR = 0xFFFFFFFF;
+        internal static uint ColorBKCOLOR = 0xFFFFFFFF;
```

Comment:
For simplicity we'll just make them public static. Your code change implies they can be set per WpfImeKeyboardHandler instance which isn't the case. 

http://cefsharp.github.io/api/83.4.x/html/M_CefSharp_Cef_ColorSetARGB.htm can be used to create the colour uint

Reply:
I intended to set both the ime handler and color within a limited range, but I respect your opinion.
Fixed.